scripts/bump-cfnspec.sh

```diff
@@ -7,7 +7,7 @@ if [ ! -f "lerna.json" ]; then
 fi
 
 ./install.sh
+./build.sh
```

Comment:
This is going to take forever!

Other suggestion: do `buildup` from `cfn2ts`? That should include `cfnspec`.

Reply:
That's not going to work. 

Once `cfn2ts` is built; it expects `@aws-cdk/core` to be built (while running a build on the newly created library). Following that it expects [decdk to be built](https://github.com/aws/aws-cdk/blob/bed53578df64b6a77c6a6e434c235ef4aefde300/packages/%40aws-cdk/cfnspec/build-tools/create-missing-libraries.ts#L281).

At this point, aren't we building everything?

Reply:
:(

Reply:
Hmm I've shipped this but does it even need to do a validating build? The validating build (that builds core and decdk and everything else) will be run as part of the PR build, no?

Only thing we need to do here is make sure that build WOULD have succeded.

Reply:
I've re-done the implementation and addressed your feedback 😊